ShopAppTests/App/Account/ViewModels/AddressFormViewModelSpec.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  AddressFormViewModelSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 2/26/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import RxSwift
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class AddressFormViewModelSpec: QuickSpec {
+    override func spec() {
+        let repositoryMock = PaymentsRepositoryMock()
+        let countriesUseCaseMock = CountriesUseCaseMock(repository: repositoryMock)
+        let viewModel = AddressFormViewModel(countriesUseCase: countriesUseCaseMock)
+        
+        describe("when view model initialized") {
+            it("should have variables with correct initial values") {
+                expect(viewModel.address).to(beNil())
+            }
+        }
+        
+        describe("when countries loaded") {
+            var disposeBag: DisposeBag!
+            
+            beforeEach {
+                disposeBag = DisposeBag()
+            }
+            
+            it("should return correct contry names") {
+                viewModel.namesOfCountries
+                    .subscribe(onNext: { names in
+                        expect(names) == ["Country1", "Country2"]
```

Comment:
Use `toEventually` method to compare values in `subscribe` closures if this subject changes after subscription.